src/CppUTest/SimpleString.cpp

```diff
@@ -194,7 +193,7 @@ size_t SimpleString::count(const SimpleString& substr) const
 {
     size_t num = 0;
     char* str = buffer_;
-    while ((str = StrStr(str, substr.buffer_))) {
+    while (*str && (str = StrStr(str, substr.buffer_))) {
```

Comment:
strstr returns the same string as it passed. That would suggest that the count would be equal to size(). What do you think about that? Which means the *str shouldn't be needed....